### DISCUSSION TITLE
Commissioner Joiner system viability

### DISCUSSION THREAD

**Discussion by Glenyz** [openthread/openthread#10875] [Q&A] [answered]
Hi,

I have been testing Openthread's commissioning protocol and have noticed significant time is needed for a joiner device to join the network relative to the specs I need to meet.

I am trying to get over 200+ devices to form a thread network within a short period of time, give or take around 1-2 minutes and it doesn't seem that the commissioner protocol will allow such specs to be met due to the nature of the commissioning process. From what I understand, the commissioner can only handle one joiner device at a time which leads to long processing time when needing to handle hundreds of devices.

Is using the commissioner approach viable for my project or should I abandon this idea and skip this authentication step?

**Comment by jwhui** [comment, marked as answer]
Given the current data requirements for Thread Commissioning (which involves establishing a DTLS session) and current PHY data rates (250 kbps), I don't think commissioning 200+ devices within 1-2 minutes is feasible.

**Reply by Glenyz** [reply]
That's unfortunate to hear, thanks for the response though.

**Reply by Glenyz** [reply]
Hi jwhui,

Sorry for reopening the discussion, but is there a way to get an estimate of how long a single joiner device may take on average to be commissioned into the network.
